src/pages/Devices/components/EditDeviceModal/EditDeviceModal.tsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useEditDeviceMutation } from 'app/api/devicesApiSlice';
+import DeviceModalForm from 'pages/Devices/components/DeviceModalForm';
+import { Device, DeviceRequestData, DeviceType, ResponseStatus } from 'types';
+import useSnackbarMessages from './useSnackbarMessages';
+
+interface Props {
+    open: boolean;
+    device?: Device;
+    onClose: () => void;
+}
+
+const EditDeviceModal = ({ open, device, onClose }: Props) => {
+    const { t } = useTranslation();
```

Comment:
Why one time `t` is created in component, and other time is passed down with `withTranslation` HOC?